.travis.yml

```diff
@@ -1,27 +1,36 @@
-language: python
-python:
-  - "2.7"
-  - "3.3"
-  - "3.4"
+# The only current way of using OS X virtual machines in Travis CI
+# is setting up the environment for Objective-C
+# This is not a problem because Python will be installed through conda
+
+language: objective-c
```

Comment:
As I've told you, all about testing and coverage is a big mess when you have to deal with the paths. The proper way, as said in the article, is to setup the package locally in editable mode. Running pytest-cov is even worse, the proper way of invoking it is a little verbose.

So what you had before:
```
py.test -vv
NUMBA_DISABLE_JIT=1 py.test --cov poliastro
```
Becomes:
```
pip install -e .
py.test poliastro -vv
NUMBA_DISABLE_JIT=1 py.test --cov poliastro poliastro/tests poliastro/twobody/tests
```

That way, **py.test** is able to find and run all the tests without `__init__.py` files inside tests folders (what aren't actually packages belonging to the project).